src/lib/kclCommands.ts

```diff
@@ -170,69 +171,107 @@ export function kclCommands(commandProps: KclCommandConfig): Command[] {
       },
     },
     {
-      name: 'open-kcl-example',
-      displayName: 'Open sample',
-      description: 'Imports an example KCL program into the editor.',
+      name: 'load-external-model',
+      displayName: 'Load external model',
+      description: 'Loads an external from a source into the current project.',
```

Comment:
Might need some rewording?

```suggestion
      description: 'Loads a model from an external source into the current project.',
```